### PR TITLE
[Feature] Support Metadata mock integration

### PR DESCRIPTION
**Pull request by zhangmo8**
### Purpose

Mock Server Docs & Mock Web Server

Metadata mock integrated

image

image

### CODE REVIEW

File: paimon-web-ui-new/mock/modules/metadata.js [hunk 1]
```diff
@@ -15,10 +15,10 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License. */
 
-module.exports = {
-  '/catalog/getAllCatalogs': {
+module.exports = (mockUtil) => ({
+  '/catalog/getAllCatalogs': mockUtil({
     code: 200,
-    message: 'success',
+    msg: 'Successfully',
```
Comment (SteNicholas) [inline paimon-web-ui-new/mock/modules/metadata.js:21]:
```suggestion
    msg: 'Success',
```

File: paimon-web-ui-new/mock/modules/metadata.js [hunk 2]
```diff
@@ -31,5 +31,116 @@ module.exports = {
         hiveConfDir: undefined
       }
     ]
-  }
-}
+  }),
+  'get /database/list': mockUtil({
+    code: 200,
+    msg: 'Successfully',
```
Comment (SteNicholas) [inline paimon-web-ui-new/mock/modules/metadata.js:37]: Ditto.

**Review by SteNicholas — COMMENTED** [review]
@zhangmo8 thanks for contribution. I left the minor comments for this pull request. PTAL.

File: paimon-web-ui-new/mock/modules/metadata.js (hunk 1 above)
Comment (zhangmo8) [reply paimon-web-ui-new/mock/modules/metadata.js:21]: This is the same message as the backend result.

**Pull request merged**